sdk/src/main/java/com/meniga/sdk/models/challenges/MenigaChallenge.java

```diff
@@ -425,12 +427,13 @@ public void merge(MenigaChallenge other) {
 	}
 
 	/**
-	 * Gets all challenges, expired, suggested and accepted alike but not expired ones
+	 * Gets all challenges - expired, suggested, accepted and disabled.
 	 *
 	 * @return A task that includes all the challenge objects that match the default parameters
 	 */
 	public static Result<List<MenigaChallenge>> fetch() {
```

Comment:
Should we keep the original functionality in a deprecated method?

Reply:
This method description was misleading - it also included expired ones (passing includeExpired: true).

But a new parameter was introduced - `includeDisabled` so this method returns all the challanges except disabled ones.

I guess this one should retrieve all the challanges by default?

Reply:
Yes I think so, the fetch with no parameters should get all items.